frameworks/java/prewarm.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "loading" > /workspace/prewarm.status
+
+# prewarm by starting the gradle daemon. Running an initial test build will also speed things up a bit
+cd /runner/frameworks/java && gradle --daemon --offline test
```

Comment:
Yeah honestly I don't even fully understand why we haven't run out of memory as it is, considering all of the containers we have and Groovyserv should take up a lot of memory as well, and its running pretty smoothly so far. I think Docker is able to manage RAM pretty well but I haven't dug into the inner workings much. It may be the reason why we have the occasional timeouts though. 

If I have to I may have to bump the ram up to t2.small. I would like to get this up in preview though and do some real world testing.